po/es.po

```diff
@@ -6536,46 +6537,50 @@ msgstr ""
 
 #: src/ownership/move-semantics.md:14
 msgid "The assignment of `s1` to `s2` transfers ownership."
-msgstr "La asignación de `s1` a `s2` transfiere el _ownership_."
+msgstr "La asignación de `s1` a `s2` transfiere la _ownership_."
 
 #: src/ownership/move-semantics.md:15
-#, fuzzy
 msgid "When `s1` goes out of scope, nothing happens: it does not own anything."
-msgstr "Cuando `s1` sale del ámbito, no sucede nada: no tiene dueño."
+msgstr ""
+"Cuando `s1` queda fuera del ámbito, no ocurre nada: no le pertenece nada."
 
 #: src/ownership/move-semantics.md:16
 msgid "When `s2` goes out of scope, the string data is freed."
-msgstr "Cuando `s2` sale del ámbito, el dato del string es liberado."
+msgstr "Cuando `s2` sale del ámbito, los datos de la cadena se liberan."
 
 #: src/ownership/move-semantics.md:17
 msgid "There is always _exactly_ one variable binding which owns a value."
-msgstr ""
-"Allí siempre hay _exactamente_ una variable enlazada que es dueña del valor."
+msgstr "Siempre hay _exactamente_ un enlace a variable que posee un valor."
 
 #: src/ownership/move-semantics.md:21
 msgid ""
 "Mention that this is the opposite of the defaults in C++, which copies by "
 "value unless you use `std::move` (and the move constructor is defined!)."
 msgstr ""
-"Hay que destacar que el contrario de los defaults en C++, que copia por "
-"valor a menos que uses `std::move` (y el movimiento del constructor está "
-"definido!)."
+"Menciona que es lo contrario de los valores predeterminados de C++, que se "
+"copian por valor, a menos que utilices `std::move` (y que el constructor de "
+"movimiento esté definido)."
 
 #: src/ownership/move-semantics.md:23
 msgid ""
 "It is only the ownership that moves. Whether any machine code is generated "
 "to manipulate the data itself is a matter of optimization, and such copies "
 "are aggressively optimized away."
 msgstr ""
+"Solo se mueve la _ownership_. Si se genera algún código máquina para "
+"manipular los datos en sí, se trata de una cuestión de optimización, y esas "
+"copias se optimizan de forma agresiva."
 
 #: src/ownership/move-semantics.md:25
 msgid ""
 "Simple values (such as integers) can be marked `Copy` (see later slides)."
 msgstr ""
+"Los valores simples (como los enteros) se pueden marcar como `Copy` "
+"(consulta las diapositivas posteriores)."
 
 #: src/ownership/move-semantics.md:27
 msgid "In Rust, clones are explicit (by using `clone`)."
-msgstr "En Rust los clones son explícitos (usando `clone`)."
+msgstr "En Rust, los clones son explícitos (si usas `clone`)."
```

Comment:
"En Rust, la clonación es explícita (usando `clone`)."

Or: "al usar `clone`"

"los clones" sounds like you have several things cloned and you are referring to the actual cloned values to be explicit not to the action.